src/kaiteki/lib/fediverse/model/emoji/emoji.dart

```diff
@@ -42,9 +48,16 @@ class CustomEmoji extends Emoji {
     required this.url,
   });
 
+  factory CustomEmoji.parse(String tag, Uri url) {
```

Comment:
No idea whether this makes sense here or elsewhere, but the pattern seems to show up in various places so...